sym_object.py

```diff
@@ -5,31 +5,55 @@
 from PySide2.QtGui import *
 from gui_views import state
 from m5_calls import *
+import copy
 
 
 class SymObject(QGraphicsItemGroup):
 
     def __init__(self, x, y, width, height, scene, component_name, name,
                     loadingFromFile, state):
         super(SymObject, self).__init__()
+
+        """common variables"""
```

Comment:
`"""` should only be used for docstrings, not for comments.

These should be normal python comments (`#`).

Let me know if I'm wrong (e.g., point me to what style you're following with this).